JS/Planes/PassengerPlane.js

```diff
@@ -2,21 +2,18 @@ const Plane = require('./Plane');
 
 class PassengerPlane extends Plane {
 
-    //=================CONSTRUCTORS=================
-
     constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, passengersCapacity) {
         super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
         this._passengersCapacity = passengersCapacity;
     }
 
-    //=================METHODS=================
     getPassengersCapacity() {
         return this._passengersCapacity;
     }
 
-    // set passengersCapacity(value) {
-    //     this._passengersCapacity = value;
-    // }
+    setPassengersCapacity(value) {
+        this._passengersCapacity = value;
+    }
```

Comment:
This code can be deleted. It is not used anywhere in our program.

Reply:
Done